sql/core/src/test/resources/sql-tests/inputs/pgSQL/int4.sql

```diff
@@ -0,0 +1,179 @@
+--
+-- Portions Copyright (c) 1996-2019, PostgreSQL Global Development Group
+--
+--
+-- INT4
+-- https://github.com/postgres/postgres/blob/REL_12_BETA1/src/test/regress/sql/int4.sql
+--
+
+CREATE TABLE INT4_TBL(f1 int) USING parquet;
+
+-- [SPARK-28023] Trim the string when cast string type to other types
+INSERT INTO INT4_TBL VALUES (trim('   0  '));
+
+INSERT INTO INT4_TBL VALUES (trim('123456     '));
+
+INSERT INTO INT4_TBL VALUES (trim('    -123456'));
+
+-- [SPARK-27923] Invalid input syntax for integer: "34.5" at PostgreSQL
+-- INSERT INTO INT4_TBL(f1) VALUES ('34.5');
+
+-- largest and smallest values
+INSERT INTO INT4_TBL VALUES ('2147483647');
+
+INSERT INTO INT4_TBL VALUES ('-2147483647');
+
+-- bad input values -- should give errors
+-- INSERT INTO INT4_TBL(f1) VALUES ('1000000000000');
+-- INSERT INTO INT4_TBL(f1) VALUES ('asdf');
+-- INSERT INTO INT4_TBL(f1) VALUES ('     ');
+-- INSERT INTO INT4_TBL(f1) VALUES ('   asdf   ');
+-- INSERT INTO INT4_TBL(f1) VALUES ('- 1234');
+-- INSERT INTO INT4_TBL(f1) VALUES ('123       5');
+-- INSERT INTO INT4_TBL(f1) VALUES ('');
+
+select * from INT4_TBL;
```

Comment:
This does not exist in the original PostgreSQL test suite.

Reply:
Removed it.